homeassistant/components/isy994/binary_sensor.py

```diff
@@ -184,7 +184,7 @@ def _detect_device_type_and_class(node: Group | Node) -> (str, str):
     # Z-Wave Devices:
     if node.protocol == PROTO_ZWAVE:
         device_type = f"Z{node.zwave_props.category}"
-        for device_class in [*BINARY_SENSOR_DEVICE_TYPES_ZWAVE]:
+        for device_class in (*BINARY_SENSOR_DEVICE_TYPES_ZWAVE,):
```

Comment:
Same here, not sure what was going on to begin with? The only reason I can imagine people want to copy it is if they are mutating it, but that's not the case here??

Reply:
```suggestion
        for device_class in BINARY_SENSOR_DEVICE_TYPES_ZWAVE:
```

Reply:
> Same here, not sure what was going on to begin with? The only reason I can imagine people want to copy it is if they are mutating it, but that's not the case here??

Could also be an artifact. I tend to use list / tuple unpacking if I want to iterate over multiple lists in one go. Something like this:
```py
for item in (*list1, *list2, some_other_item):
   ...
```
Only the other list was never added.

Reply:
Found the reason. `BINARY_SENSOR_DEVICE_TYPES_ZWAVE` is a `dict`. Unpacking will thus only create a list over all keys. It's actually even better to use `.items()` here instead.

Reply:
The default iterator of a dict is the keys, there is no reason to convert it first.

Reply:
> The default iterator of a dict is the keys, there is no reason to convert it first.

Yeah. Whoever added it didn't though of that I imagine.